docs/source/create/entities/create-fountain.rst

```diff
@@ -0,0 +1,93 @@
+#####################################
+Tutorial: Create a Smoke Fountain
+#####################################
+
+In this tutorial, you will learn how particle entities work by creating a smoke fountain that emits multiple colors. Particle entities are used to create effects made up of many small particles, such as fire, smoke, or falling leaves.
+
+.. contents:: On This Page
+    :depth: 2
+
+
+--------------------------------------
+Prerequisites
+--------------------------------------
+
+Consider getting familiar with the following concepts before starting this tutorial:
+
++ `Create New Entities <create-entities.html>`_
++ `Change How Entities Look <entity-appearance.html>`_
+
+
+---------------------------------
+Create a Smoke Fountain
+---------------------------------
+
+Particle entities are used to create effects that are made up of smaller parts such as smoke, fire, or swirling water. You can create any effect of your choice using different textures. The default texture for all particle entities is a `wispy smoke texture <https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png>`_. You can edit other particle entity properties or replace the texture to create your desired effect. We support textures in the PNG and JPEG formats.
```

Comment:
Took out the last sentence b/c i think we support a lot more textures now.

```suggestion
Particle entities are used to create effects that are made up of smaller parts such as smoke, confetti, or falling leaves. The entity's effect and appearance is defined by its texture. The default texture is a `wispy smoke texture <https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png>`_, but you can replace this texture with your own to create your desired effect. 
```

Reply:
I'll include the new types of supported textures to the PBR Materials page as well.